README.md

```diff
@@ -22,5 +24,11 @@ _--verbose_: A flag used to output result metrics
 lightning schedule <url> --interval --start
 ```
 _url_: valid url of the website you want to be analyzed.
+
 _--interval_: A time interval in milliseconds. Defaults to one day. Minimum value 5 minutes (300000)
-_--start_: A milliseconds timestamp specifying when the scanning starts. Defaults to now + interval
+
+_--start-stamp_: A milliseconds timestamp specifying when the scanning starts. Defaults to now + interval
+
+_--every_: An optional argument to set interval in a friendly way. Will override --interval. Example: --every 2-hours.
```

Comment:
Maybe add an expected format along with the example so the user understands what the error is. Maybe link to http://momentjs.com/docs/#/durations/